gapic/src/main/com/google/gapid/perfetto/views/CpuPanel.java

```diff
@@ -67,7 +67,7 @@ public CpuPanel copy() {
 
   @Override
   public String getTitle() {
-    return "CPU " + (track.getCpu() + 1);
```

Comment:
Is there a specific reason why you want CPU ids to show up as their id + 1? I would prefer to show CPUs as their actual ids (typically starting at 0).

Reply:
The IDs are rather arbitrary and it makes more sense to start counting at 1 when displaying.

Reply:
Are they really arbitrary? I think they are the IDs in the ftrace data that is collected by traced_probes. Also, if you want consistency with perfetto.dev, the actual IDs should be used.

Reply:
At least on Linux, the CPU ids used directly map to the ids in the ftrace filesystem; see for example https://android.googlesource.com/platform/external/perfetto/+/refs/heads/master/src/traced/probes/ftrace/ftrace_procfs.cc

Reply:
Done.